ci/release/update-version.sh

```diff
@@ -45,3 +45,7 @@ sed_runner 's/'"branch-.*\/RAPIDS.cmake"'/'"branch-${NEXT_SHORT_TAG}\/RAPIDS.cma
 for FILE in conda/environments/*.yml; do
   sed_runner "s/cudf=${CURRENT_SHORT_TAG}/cudf=${NEXT_SHORT_TAG}/g" ${FILE};
 done
+
+# Doxyfile update
+sed_runner "s|\(TAGFILES.*librmm/\).*|\1${NEXT_SHORT_TAG}|" cpp/doxygen/Doxyfile
+sed_runner "s|\(TAGFILES.*libcudf/\).*|\1${NEXT_SHORT_TAG}|" cpp/doxygen/Doxyfile
```

Comment:
```suggestion
sed_runner "/TAGFILES/ s|[0-9]\+.[0-9]\+|${NEXT_SHORT_TAG}|g" cpp/doxygen/Doxyfile
```

The original changes here didn't work for me when I tested them locally. This suggested change seems to do the trick. You can verify this locally by running `bash ci/release/update-version.sh 22.10.00` and verifying the results.